### PR TITLE
Int: Change Runtime CI - remove pre-generated JSONs and add pre-test step to generate them on flight

### PR DESCRIPTION
**Pull request by SebastianMorawiec**
# The problem
https://zapatacomputing.atlassian.net/browse/ORQSDK-1040
We had to keep the generated IRs

# This PR's solution
Generate IRs in flights before testing starts

# Checklist

_Check that this PR satisfies the following items:_

- [x] Tests have been added for new features/changed behavior (if no new features have been added, check the box).
- [x] The [changelog file](CHANGELOG.md) has been updated with a user-readable description of the changes (if the change isn't visible to the user in any way, check the box).
- [x] The PR's title is prefixed with `<feat/fix/chore/imp[rovement]/int[ernal]/docs>[!]:`
- [x] The PR is linked to a JIRA ticket (if there's no suitable ticket, check the box).

### CODE REVIEW

File: projects/orquestra-sdk/Makefile [hunk 1]
```diff
@@ -33,6 +33,8 @@ test:
 # Min code-test coverage measured for the whole project required for CI checks to pass.
 MIN_COVERAGE=75
 
+# Nothing to do for this repo
```
Comment (alexjuda) [inline projects/orquestra-sdk/Makefile:36]:
```suggestion
# Nothing to do for this subproject
```

File: projects/orquestra-workflow-runtime/Makefile [hunk 2]
```diff
@@ -29,6 +29,15 @@ test:
 MIN_COVERAGE=75
 
 
+# We do not want to keep the IRs in json form in the repo as maintanance is a big hussle
+# However we want to make sure that during main testing we do not have SDK installed
+pre-coverage:
+	$(PYTHON) -m pip install -e '../orquestra-sdk'
+	$(PYTHON) -m pip install -e '../orquestra-workflow-shared'
+	cd tests/data; $(PYTHON) generate_workflow_defs.py
+	$(PYTHON) -m pip uninstall -y orquestra-sdk
```
Comment (alexjuda) [inline projects/orquestra-workflow-runtime/Makefile:38]: Have you considered making this part of `make github_actions` in `orquestra-workflow-runtime`?

**Review by alexjuda — APPROVED** [review]
Non-blocking suggestion

File: projects/orquestra-workflow-runtime/Makefile (hunk 2 above)
Comment (SebastianMorawiec) [reply projects/orquestra-workflow-runtime/Makefile:38]: Yes, I just believe that the separation of steps between:
"Here we prepare our tests"
"Here we run our tests" 
is good practice, thus 2 steps are better here IMHO

**Pull request merged**